app/src/main/java/at/bitfire/icsdroid/model/EditSubscriptionModel.kt

```diff
@@ -29,7 +36,8 @@ class EditSubscriptionModel(
     var uiState by mutableStateOf(UiState())
         private set
 
-    val subscriptionWithCredential = db.subscriptionsDao().getWithCredentialsByIdLive(subscriptionId)
+    val subscriptionWithCredential = db.subscriptionsDao().getWithCredentialsByIdFlow(subscriptionId)
+        .stateIn(viewModelScope, SharingStarted.Eagerly, null)
```

Comment:
Any specific reason to use `SharingStarted.Eagerly` ? I think we should always prefer `SharingStarted.WhileSubscribed(5000)` if possible, which seems to be the case here.